doc/ale-dart.txt

```diff
@@ -27,6 +27,17 @@ g:ale_dart_analysis_server_executable   *g:ale_dart_analysis_server_executable*
   This variable can be set to change the path of dart.
 
 
+g:ale_dart_analysis_server_enable_language_server
+                            *g:ale_dart_analysis_server_enable_language_server*
+                            *b:ale_dart_analysis_server_enable_language_server*
+  Type: |Number|
+  Default: `1`
+
+  When set to `1`, ALE will use the new `dart language-server` command to
+  launch the language server. When set to `0`, ALE will use the original
+  `./snapshots/analysis_server.dart.snapshot --lsp` command.
+
```

Comment:
Maybe adding the dart version from where the new command is necessary would help people decide if they need to change this flag or not. Exampl:

```
When set to `1`, ALE will use the new `dart language-server` command, 
available from Dart version XX.XX.XX to launch the language server. When
set to `0`, ALE will instead use the deprecated `./snapshots/analysis_server.dart.snapshot --lsp`
command used by older versions of Dart.

Reply:
I have updated the documentation to state that dart version `2.16.0` is when the new command was released.

* https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#2160---2022-02-03
* https://github.com/dart-lang/sdk/blob/2.16.0/pkg/analysis_server/tool/lsp_spec/README.md#running-the-server
* https://github.com/dart-lang/sdk/blob/2.15.1/pkg/analysis_server/tool/lsp_spec/README.md#running-the-server